module.prop

```diff
@@ -1,7 +1,7 @@
 id=Pix3lify
 name=Pix3lify
-version=v2.2
-versionCode=220
+version=v2.2.11
+versionCode=2211
```

Comment:
We can really just increment ``v2.2.1`` to ``v2.2.2``, as specified by semver

Reply:
Yes, but if we do testing we have to bump

Reply:
Actually you are correct

Reply:
:)